web/docs/setup/monitor.md

```diff
@@ -71,14 +71,50 @@ vast:
       real-time: false
       path: /tmp/vast-metrics.sock
       type: datagram # possible values are "stream" or "datagram"
+    # Configures if and where metrics should be written to VAST itself.
+    self-sink:
+      enable: false
 ```
 
-Both sinks write metrics as NDJSON and inlines the `metadata` key-value pairs
-into the top-level object. VAST buffers metrics for these sinks in order to
-batch I/O operations. To enable real-time metrics reporting, set the options
-`vast.metrics.file-sink.real-time` or `vast.metrics.uds-sink.real-time`
+Both the file and UDS sinks write metrics as NDJSON and inlines the `metadata`
+key-value pairs into the top-level object. VAST buffers metrics for these sinks
+in order to batch I/O operations. To enable real-time metrics reporting, set the
```

Comment:
```suggestion
to batch I/O operations. To enable real-time metrics reporting, set the
```

Reply:
Fixed this in https://github.com/tenzir/vast/pull/2976/commits/f594483ae0a48c67bb844cee77dc7294b781aaf6 already.